x-pack/plugins/security_solution/server/lib/entity_analytics/risk_score/risk_score_data_client.test.ts

```diff
@@ -452,4 +454,37 @@ describe('RiskScoreDataClient', () => {
       }
     });
   });
+  describe('upgrade process', () => {
+    beforeEach(() => {
+      jest.spyOn(riskScoreDataClient, 'upgrade');
+      spyOnPrivateMethod(riskScoreDataClient, 'upsertRiskScoreLatestIndex');
```

Comment:
These tests are now outdated based on recent changes, need to update them...

Reply:
Fixed: https://github.com/elastic/kibana/pull/174474/commits/63e4fb943f46e875207f59d61d4970bcf069f3f6